deploy.yml

```diff
@@ -58,6 +58,76 @@ resources:
   source:
     url: {{slack_webhook_url}}
 
+- name: eq-author-release
+  type: github-release
+  source:
+    owner: ONSdigital
+    repository: eq-author
+    access_token: ((github_access_token))
+    release: true
+    pre_release: true
```

Comment:
I think we should drop `pre_release` from both these author releases and runner ones too